dhall-json/src/Dhall/JSON.hs

```diff
@@ -319,12 +322,7 @@ dhallToJSON e0 = loop (Core.alphaNormalize (Core.normalize e0))
         Core.RecordLit a ->
             case toOrderedList a of
                 [   (   "contents"
-                    ,   Core.App
-                            (Core.Field
-                                _
-                                alternativeName
-                            )
-                            contents
+                    ,   contents
                     )
                  ,  (   "field"
```

Comment:
The additional indentation here confused my for so long, I ended up opening [a GHC issue](https://gitlab.haskell.org/ghc/ghc/issues/17028)! ;)

Reply:
Yeah, the indentation there bugged me, too 🙂